src/shared/Form/FormField.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+export default function FormField({ label, type = 'text', value, onChange, disabled = false, infoText = '', style, inputClassName, inputAddon }) {
```

Comment:
I suggest naming this component `TextInput` to be more consistent with the other component names.